Mini aod dev 74 x

fanbomeng opened the pull request:
adding electron variable for LFV analysis @mcepeda

taroni committed:
make it work for gen quantities in CMSSW74X ntuples

taroni committed:
first attempt in 74X

taroni committed:
adding more mass point for higgs

taroni committed:
2012 data scale factors

taroni committed:
2012 scalefactors

taroni committed:
2012 scalefactor

taroni committed:
2012 scale factors

taroni committed:
adding MuTau samples

fanbomeng committed:
check electron in new version
	modified:   NtupleTools/python/templates/electrons.py
	modified:   NtupleTools/python/templates/event.py

fanbomeng committed:
check on electrons from previous version

	modified:   NtupleTools/python/templates/electrons.py

bucky-badger commented:
Can one of the admins verify this patch?

bucky-badger commented:
Can one of the admins verify this patch?

bucky-badger commented:
Can one of the admins verify this patch?

bucky-badger commented:
Can one of the admins verify this patch?

nwoods commented:
Thanks Bucky, we'll take care of it.

dntaylor commented on NtupleTools/python/templates/electrons.py:
```diff
@@ -106,6 +106,16 @@
+
+
+
+    objectGenMotherPt =  '? (getDaughterGenParticleMotherSmart({object_idx}, 11, 0).isAvailable && getDaughterGenParticleMotherSmart({object_idx}, 11, 0).isNonnull) ? getDaughterGenParticleMotherSmart({object_idx}, 11, 0).pt() : -999', 
```
Just an organizational thing: can you move this up to https://github.com/uwcms/FinalStateAnalysis/blob/miniAOD_dev_74X/NtupleTools/python/templates/electrons.py#L92 so its with the rest of the gen stuff?

And also add it to muons here: https://github.com/uwcms/FinalStateAnalysis/blob/miniAOD_dev_74X/NtupleTools/python/templates/muons.py#L57

taroni committed:
try to analyse 13 TeV data

taroni committed:
try to analyse 13 TeV data

fanbomeng committed:
svfit
	modified:   DataFormats/interface/PATFinalState.h
	modified:   DataFormats/src/PATFinalState.cc
	modified:   MetaData/python/data13TeV.py
	modified:   NtupleTools/python/ntuple_builder.py
	modified:   NtupleTools/python/templates/topology.py
	modified:   NtupleTools/test/make_ntuples_cfg.py
	modified:   PatTools/python/patFinalStateProducers.py

fanbomeng committed:
for LVFETAU
	new file:   TagAndProbe/python/HetauCorrection.py
	new file:   TagAndProbe/python/correctionloader.py

fanbomeng committed:
change for SFfitLFV
	modified:   NtupleTools/python/ntuple_builder.py
	modified:   NtupleTools/test/make_ntuples_cfg.py
	modified:   Utilities/python/struct.py

fanbomeng committed:
svfitLFV

taroni committed:
fakerate fit and condor utilities working with 74X

taroni committed:
merging 74X_SLC6

taroni committed:
using datadef for LFV only

fanbomeng committed:
Merge branch 'miniAOD_dev_74X' of https://github.com/taroni/FinalStateAnalysis into miniAOD_dev_74X

Conflicts:
	NtupleTools/python/templates/electrons.py
	NtupleTools/test/make_ntuples_cfg.py
update for fake

	modified:   DataAlgos/src/helpers.cc
	modified:   DataFormats/src/PATFinalState.cc
	new file:   MetaData/python/data13TeV_LFV.py
	modified:   MetaData/python/data_views.py
	modified:   MetaData/scripts/get_mc_lumi.py
	new file:   MetaData/tuples/MiniAOD-13TeV.json
	modified:   MetaData/tuples/MiniAOD-13TeV_PHYS14DR.json
	modified:   MetaData/tuples/MiniAOD-13TeV_RunIISpring15DR74.json
	modified:   NtupleTools/python/ntuple_builder.py
	modified:   NtupleTools/python/templates/candidates.py
	modified:   NtupleTools/python/templates/cleaning.py
	modified:   NtupleTools/python/templates/electrons.py
	modified:   NtupleTools/python/templates/event.py
	modified:   NtupleTools/python/templates/taus.py
	modified:   NtupleTools/python/templates/trigger.py
	modified:   PlotTools/rake/cython.rake
	modified:   PlotTools/rake/meta.rake
	modified:   PlotTools/scripts/discover_ntuples.py
	modified:   StatTools/scripts/fit_efficiency_chi2.py
	new file:   TagAndProbe/data/EfficiencySingleElectron_MySelToHLT_efficiency.txt
	new file:   TagAndProbe/data/ScaleFactors_MySelToHLT_efficiency.txt
	new file:   TagAndProbe/data/efficiency-Run2012ABCD-WP80ToHLTEle.txt
	new file:   TagAndProbe/data/electronEmbeddedScaleFactor.txt
	new file:   TagAndProbe/data/scaleFactor-Run2012ABCD-GsfElectronToId.txt
	new file:   TagAndProbe/data/scaleFactor-Run2012ABCD-RecoToIso.txt
	new file:   TagAndProbe/data/scaleFactor-Run2012ABCD-SCToElectron.txt
	modified:   TagAndProbe/python/H2TauCorrections.py
	modified:   Utilities/scripts/condor_summary.py

Commit:
First commit related to LFV SVFit (Not ready to be used yet!)

fanbomeng committed:
Merge branch 'miniAOD_dev_74X' of https://github.com/mcepeda/FinalStateAnalysis into miniAOD_dev_74X

Conflicts:
	DataFormats/src/PATFinalState.cc

svfit
	modified:   DataAlgos/BuildFile.xml
	new file:   DataAlgos/interface/ApplySVfitLFV.h
	new file:   DataAlgos/src/ApplySVfitLFV.cc
	modified:   DataFormats/interface/PATFinalState.h
	modified:   DataFormats/src/PATFinalState.cc
	modified:   NtupleTools/python/templates/topology.py
	modified:   StatTools/python/RooFunctorFromWS.py
	modified:   StatTools/scripts/fit_efficiency_chi2.py

fanbomeng committed:
Merge branch 'miniAOD_dev_74X' of github.com:uwcms/FinalStateAnalysis into miniAOD_dev_74X
update to online

taroni committed:
adding clone method to struct.py

fanbomeng committed:
Merge branch 'miniAOD_dev_74X' of github.com:uwcms/FinalStateAnalysis into miniAOD_dev_74X
update to pileup correction

taroni committed:
make plots of 13TeV analysis

fanbomeng committed:
update for plotter

fanbomeng committed:
muon uncertainty
